packages/react-table/src/components/Table/Table.tsx

```diff
@@ -30,6 +30,12 @@ export enum TableVariant {
   compact = 'compact'
 }
 
+export enum SelectedRowsAmount {
```

Comment:
Do you think we could assign strings to represent these enum values? I think using the enum indices is really elegant but makes it a little difficult to debug when comparing against the resulting markup. In other words I thought `data-rowsamount="1"` meant there was a single row selected instead of it referring to "some".